config-linux.md

```diff
@@ -107,6 +111,17 @@ Note that the number of mapping entries MAY be limited by the [kernel][user-name
 ]
 ```
 
+## <a name="configLinuxTimeOffset" />Offset for Time Namespace
+
+**`timeOffsets`** (object, OPTIONAL) sets the offset for Time Namespace. For more information
+see the [time_namespaces](time_namespaces.7).
+
+The name of the clock is the entry key.
+Entry values are objects with the following properties:
+
+* **`secs`** *(int64, OPTIONAL)* - is the offset of clock (in secs) in the container.
+* **`nanosecs`** *(uint32, OPTIONAL)* - is the additional offset for secs (in nanosecs). The actual offset is secs plus nanosecs.
```

Comment:
> The actual offset is secs plus nanosecs.

Just to confirm, is this true for negative secs too?

Also curious why nanosecs is unsigned

Reply:
updated

Reply:
Sorry for going back and forth, but nanosecs can remain unsigned, as it is stated in the kernel man page
https://man7.org/linux/man-pages/man7/time_namespaces.7.html

> These offsets are expressed
       relative to the clock values in the initial time namespace.  The
       offset-secs value can be negative, subject to restrictions noted
       below; offset-nanosecs is an unsigned value.

Reply:
No problem. I re-updated to remain unsigned.